client/src/components/molecules/Choice.tsx

```diff
@@ -0,0 +1,72 @@
+import { theme } from "@/theme";
+import styled from "@emotion/styled";
+import type { Choice } from "biseo-interface/agenda";
+import { Box, Text } from "../atoms";
+import { ReactComponent as SelectIcon } from "@/assets/select.svg";
+
+const ChoiceContainer = styled.div<{
+  chosen?: boolean;
+}>`
+  border-radius: 5px;
+  background-color: ${props =>
+    props.chosen ? props.theme.colors.blue600 : props.theme.colors.white};
+  border: 1px solid ${props => props.theme.colors.gray200};
+  padding: 6px 13px 6px 13px;
+  width: 340px;
+  height: 30px;
+  gap: 10px;
+`;
+
+type ChoiceProps = {
+  choice: Choice;
+  chosen: boolean;
+  onClick: () => void;
+};
+
+const ChoiceComponent: React.FC<ChoiceProps> = ({
+  choice,
+  chosen,
+  onClick,
+}) => {
+  const iconColor = chosen ? theme.colors.blue600 : theme.colors.gray500;
+  const textColor = chosen ? "white" : "gray500";
+
+  return (
+    <ChoiceContainer onClick={onClick} chosen={chosen}>
+      <Box dir="row" gap={10}>
+        <SelectIcon stroke={iconColor} />
+        <Text variant="body" color={textColor}>
+          {choice.name}
+        </Text>
+      </Box>
+    </ChoiceContainer>
+  );
+};
+
+const CompletedChoice: React.FC = () => {
+  return (
+    <ChoiceContainer onClick={() => null} chosen={true}>
+      <Box dir="row" gap={10}>
+        <SelectIcon stroke={theme.colors.blue600} />
+        <Text variant="body" color="white">
+          투표 완료
+        </Text>
+      </Box>
+    </ChoiceContainer>
+  );
+};
```

Comment:
이런 식으로 내부에 logic이 없는 (그리고 앞으로도 없을 예정인) 컴포넌트의 경우 `return ( ... )`을 쓰지 않고 arrow function으로 바로 return(?) 시키면 code의 visual한 면에서도 "아 이 컴포넌트는 로직이 없는 pure한 component구나"를 알 수 있어요! 그리고 indentation도 한 단계 줄일 수 있어서 좋아요 (라고 어제 순호한테 배웠음)
다른 파일들도 쭉 보면서 이런 케이스 있다면 수정해주시면 순호가 좋아할거에요 (+ 순호의 pr 리뷰가 짧아질거에요)
```suggestion
const CompletedChoice: React.FC = () => (
  <ChoiceContainer onClick={() => null} chosen={true}>
    <Box dir="row" gap={10}>
      <SelectIcon stroke={theme.colors.blue600} />
      <Text variant="body" color="white">
        투표 완료
      </Text>
    </Box>
  </ChoiceContainer>
);
```

Reply:
최대한 로직이 여기저기 컴포넌트에 나뉘어 있지 않도록(?) 컴포넌트를 잘 쪼개고 정리하는 게 중요하다고도 했어용